src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,74 @@
+import { Box, Container } from "@mui/material";
+import React from "react";
+import { useTranslation } from "react-i18next";
+
+import { FooterNav } from "~/components/footer/nav.tsx";
+import { SocialIcons } from "~/components/footer/social-icons.tsx";
+import {
+	FooterBottomContainer,
+	FooterContainer,
+	SocialIconsContainer,
+	StyledDivider,
+	StyledFooterCopyrightText,
+	StyledFooterText,
+	StyledFooterTextContainer,
+} from "~/components/footer/styles.ts";
+import { MainLogo } from "~/components/header/main-logo.tsx";
+import { colors } from "~/libs/constants/color.ts";
+import { footerColumnsLogIn } from "~/libs/constants/footer-links.ts";
+import {
+	socialMediaDataLogIn,
+	socialMediaDataLogOut,
+} from "~/libs/constants/social-media-icons.ts";
+
+export const Footer: React.FC = () => {
+	const { t } = useTranslation();
+	const isLoggIned = false;
```

Comment:
what is it?) no comments if its temporary, we cant left as it is, no one will remember after few days this strange logic

Reply:
done